lib/pbench/server/api/resources/datasets_metadata.py

```diff
@@ -134,16 +137,35 @@ def _put(self, json_data: JSONOBJECT, _) -> Response:
 
         # Validate the authenticated user's authorization for the combination
         # of "owner" and "access".
-        self._check_authorization(str(dataset.owner_id), dataset.access)
+        #
+        # Note that we allow authenticated clients to set "user.*" namespace
+        # metadata with READ access to the dataset because it doesn't affect
```

Comment:
You might want to update the example at lines 109-112 and 126.

Reply:
The example will still work just fine... although I suppose "contact information" isn't perhaps the best example for user-specific non-owner metadata. 😆